library/VTKExtensions/Readers/Testing/TestF3DGenericImporter.cxx

```diff
@@ -25,13 +26,17 @@ int TestF3DGenericImporter(int argc, char* argv[])
 
   // Test valid reader filepath
   std::string filename = std::string(argv[1]) + "data/small.ex2";
+  exodusReader->SetFileName(filename.c_str());
   importer->SetFileName(filename);
+  importer->SetInternalReader(exodusReader);
   if (!importer->CanReadFile())
   {
     std::cerr << "Importer unexpectedly can not read a valid file" << std::endl;
     return EXIT_FAILURE;
   }
 
+  exodusReader->UpdateInformation(); // TODO: why is it required?
```

Comment:
I'm a bit confused about this one. If I remove this line, the test is failing.

Reply:
is this specific to exudus or same behavior with other readers ?

Reply:
I'll check

Reply:
I changed the reader to `vtkGLTFReader`. We need to call `UpdateInformation` anyway to get the available animations, so this comment is not relevant anymore.